ejb/src/main/java/org/wildfly/httpclient/ejb/HttpEJBReceiver.java

```diff
@@ -283,7 +283,7 @@ protected SessionID createSession(final EJBReceiverSessionCreationContext receiv
         CompletableFuture<SessionID> result = new CompletableFuture<>();
 
         RequestBuilder builder = new RequestBuilder()
-                .setRequestType(RequestType.OPEN)
+                .setRequestType(RequestType.CREATE_SESSION)
```

Comment:
I like this - a more accurate description of the operation.

Reply:
I agree.